wp-content/civi-extensions/goonjcustom/api/v3/Goonjcustom/InstitutionCollectionCampCron.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @file
+ */
+
+use Civi\Api4\EckEntity;
+use Civi\Api4\OptionValue;
+use Civi\InstitutionCollectionCampService;
+
+/**
+ * Goonjcustom.CollectionCampCron API specification (optional)
+ * This is used for documentation and validation.
+ *
```

Comment:
It's not an `CollectionCampCron` when you are copying template from another file please update the name too